libft/tests/map/mapss/mapss_test.c

```diff
@@ -15,11 +15,13 @@
 extern int memcheck_enabled;
 void	mapss_test(void)
 {
-	ft_printf("mapss print in testing\n");
+	ft_printf("mapss: gen in testing\n");
+	ft_mapss_gen_test();
+	ft_printf("mapss: print in testing\n");
 	ft_mapss_print_test();
 	if (!memcheck_enabled)
 	{
-		ft_printf("mapss add in testing\n");
+		ft_printf("mapss: add in testing\n");
 		ft_mapss_add_test();
```

Comment:
Short answer: I don't want to, fixed in one of the next PRs
Long answer: when I introduced make fulltest as prerequisite for a merge, your part (mapss add) was failing both norminette and valgrind. I thought I told you about that, but I'm not sure alreafydy. Anyway, I introduced the check, and couldn't even merge it. So I circumvented the checks for your part, otherwise I'd had to fix everything myself.
Now that you normed that part, we could turn the full norminette check back in makefile.
Also I stumbled upon this memcheck exclusion myself somewhere in these open prs and fixed the memory issues of mapss add.
So when you go through all these prs, you'll notice this condition removed too.